packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -458,9 +450,9 @@ export class DatePickerBase extends BaseComponent<IDatePickerProps, IDatePickerS
           }
         }
       } else {
-        // No input date string shouldn't be an error if field is not required
+        // Only show error for empty inputValue if it is a required field
         this.setState({
-          errorMessage: ''
+          errorMessage: isRequired ? strings!.isRequiredErrorMessage || ' ' : ''
```

Comment:
Minor nit, but why is the blank space needed here?

Shouldn't this just be:
```ts
errorMessage: isRequired && (strings!.isRequiredErrorMessage)
```
That way it's either a message or undefined.

Reply:
I just kept it there because it was there in the previous implementation. I'll test without it

Reply:
need to keep it since the textbox will not display in error state (i.e. red border) if the fallback is not truthy.